### PR TITLE
Updated dead links in configuation-in-mattermost-database.rst

### PR DESCRIPTION
**Pull request by ArmanChand**

### CODE REVIEW

File: source/configure/configuation-in-mattermost-database.rst [hunk 1]
```diff
@@ -157,11 +157,11 @@ The command to migrate the config to the database should always be run as the *m
...
    
-As with the environment file you'll have to escape any single quotes in the database connection string. Also, any existing SAML certificates will be migrated into the database as well so they are available for all servers in the cluster.
+As with the environment file, you'll have to escape any single quote in the database connection string. Also, any existing SAML certificates will be migrated into the database as well so they are available for all servers in the cluster.
 
-With configuration in the database enabled, any changes to the configuration are recorded to the ``Configurations`` and ``ConfigurationFiles`` tables. Furthermore, ``ClusterSettings.ReadOnlyConfig`` is ignored, enabling full use of the System Console.
+With the configuration in the database enabled, any changes to the configuration are recorded to the ``Configurations`` and ``ConfigurationFiles`` tables. Furthermore, ``ClusterSettings.ReadOnlyConfig`` is ignored, enabling full use of the System Console.
```
Comment (cwarnermm) [inline source/configure/configuation-in-mattermost-database.rst:164]:
```suggestion
When configuration in the database is enabled, any changes to the configuration are recorded to the ``Configurations`` and ``ConfigurationFiles`` tables. Furthermore, ``ClusterSettings.ReadOnlyConfig`` is ignored, enabling full use of the System Console.
```

**Review by cwarnermm — COMMENTED** [review]
Thank you for this docs contribution! I've provided feedback inline.

File: source/configure/configuation-in-mattermost-database.rst (hunk 1 above)
Comment (ArmanChand) [reply source/configure/configuation-in-mattermost-database.rst:164]: updated

File: source/configure/configuation-in-mattermost-database.rst [hunk 2]
```diff
@@ -157,11 +157,11 @@ The command to migrate the config to the database should always be run as the *m
    bin/mattermost config migrate ./config/config.json 'mysql://mmuser:mostest@tcp(127.0.0.1:3306)/mattermost?charset=utf8mb4,utf8&writeTimeout=30s'
 
 .. warning::
-   When migrating config, Mattermost will incorporate configuration from any existing ``MM_*`` environment variables set in the current shell.  See `Environment Variables  <https://docs.mattermost.com/administration/config-settings.html#configuration-settings>`_
+   When migrating config, Mattermost will incorporate configuration from any existing ``MM_*`` environment variables set in the current shell.  See `Environment Variables  <https://docs.mattermost.com/configure/configuration-settings.html>`_
    
-As with the environment file you'll have to escape any single quotes in the database connection string. Also, any existing SAML certificates will be migrated into the database as well so they are available for all servers in the cluster.
+As with the environment file, you'll have to escape any single quote in the database connection string. Also, any existing SAML certificates will be migrated into the database as well so they are available for all servers in the cluster.
```
[the remark this answers is not in the document]
Comment (ArmanChand) [reply source/configure/configuation-in-mattermost-database.rst]: updated

**Comment by ArmanChand** [comment]
@cwarnermm kindly merge this pr

File: source/configure/configuation-in-mattermost-database.rst (hunk 1 above)
Comment (cwarnermm) [reply source/configure/configuation-in-mattermost-database.rst:164]: @ArmanChand - This update doesn't appear to be committed yet.

**Comment by ArmanChand** [comment]
@cwarnermm , updated kindly merge this PR

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**